+each/+iterators/AllPermutationIterator.m

```diff
@@ -10,17 +10,17 @@
     %
     % See Also: each, eachPermutation, each.iterators.Iterable
     %
-    
-    %   Copyright 2014 The MathWorks, Inc.
-    
+
+    % Copyright 2014 The MathWorks, Inc.

```

Comment:
Copyrights are out of date

Reply:
I didn't update them for whitespace only changes